admin/src/core/services/cognito.service.ts

```diff
@@ -39,7 +50,7 @@ export class CognitoService {
         return null;
       }
       return new Promise<any>((resolve) => {
-        return Auth.currentAuthenticatedUser()
+        return getCurrentUser()
```

Comment:
In Amplify v6, need to use getCurrentUser(), no currentAuthenticatedUser() anymore.